tests/jest/secrets/secrets_crud_operations.test.js

```diff
@@ -115,13 +103,7 @@ describe('Test secret crud operations', () => {
         expect(res.body).toMatchObject({
             namespace: testNamespace,
             secrets: {
-                pageInfo: {
-                    order: [],
-                    filter: [],
-                    limit: 0,
-                    offset: 0,
-                    total: 0
-                },
+                pageInfo: null,
                 results: []
             }
         })
```

Comment:
This test suite needs an extra set of tests that creates and executes a workflow that verifies the contents of a secret. That ensures nothing has been broken between the secrets store and the engine.